docs/source/managed-federation/overview.mdx

```diff
@@ -50,3 +50,6 @@ By using a configuration manager that's external to your gateway, you help ensur
 <hr/>
 
 Ready to try it out? Continue to [Setup](./setup/).
+
+## Manual Composition
+Using Managed Federation should be the default for all production systems, but in some use cases, like local developement, it can be helpful to [manualy compose the supergraph yourself](https://www.apollographql.com/docs/technotes/TN0006-unmanaged-federation/).
```

Comment:
```suggestion
**Managed federation is strongly recommended for all production environments.** For non-production environments (such as local development), it can be helpful instead to [manually compose the supergraph schema](/technotes/TN0006-unmanaged-federation/).
```